### PR TITLE
Hello World on Main Storyboard

### PR DESCRIPTION
**Pull request by SohagCSE**
added hello world on home screen storyboard

### CODE REVIEW

File: DemoPullRequest/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,24 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13122.16" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="21507" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+    <device id="retina6_12" orientation="portrait" appearance="light"/>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13104.12"/>
```
Comment (NurAhmadullah) [inline DemoPullRequest/Base.lproj/Main.storyboard:4]: make sure, no red alert there

File: DemoPullRequest/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
-        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21505"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--View Controller-->
         <scene sceneID="tne-QT-ifu">
             <objects>
```
Comment (NurAhmadullah) [inline DemoPullRequest/Base.lproj/Main.storyboard:12]: this is not good approach, make sure create UI programmatically

File: DemoPullRequest/Base.lproj/Main.storyboard (hunk 1 above)
Comment (SohagCSE) [reply DemoPullRequest/Base.lproj/Main.storyboard:4]: > make sure, no red alert there

I have fix the issue, please continue the merge

**Review by NurAhmadullah — COMMENTED** [review]
nothing to add on comment

**Pull request merged**